R/R-igraph/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           R 1.0
+
+R.setup             cran igraph igraph 1.3.5
+revision            0
+maintainers         {gmail.com:szhorvat @szhorvat} {@barracuda156 gmail.com:vital.had} openmaintainer
+license             {GPL-2 GPL-3}
```

Comment:
```suggestion
license             GPL-2+
```

Reply:
I followed what was used for R itself, I think. CRAN expands `GPL-2+` to `GPL-2 GPL-3`: https://cran.r-project.org/web/packages/igraph/index.html
But I don’t mind setting `GPL-2+` either. No preference here.